baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -206,7 +206,7 @@ export class AddressesClient {
     // Iterate over each of the methods that the service provides
     // and create an API call method for each.
     const addressesStubMethods =
-        ['aggregatedList', 'delete', 'insert', 'list'];
+        ['aggregatedList', 'unaryDelete', 'unaryInsert', 'list'];
```

Comment:
We must not rename the names in the stubs. The stub still has original names taken from the protos.